tests/integration/test_request_queue.py

```diff
@@ -1193,28 +1192,48 @@ async def test_request_queue_has_stats(request_queue_apify: RequestQueue) -> Non
     assert apify_metadata.stats.write_count == add_request_count
 
 
-async def test_rq_long_url(request_queue_apify: RequestQueue) -> None:
-    # TODO: Remove the skip when issue #630 is resolved.
-    if isinstance(request_queue_apify._client._implementation, ApifyRequestQueueSharedClient):  # type: ignore[attr-defined]
-        pytest.skip('Skipping for the "shared" request queue - unskip after issue #630 is resolved.')
-
-    url = 'https://portal.isoss.gov.cz/irj/portal/anonymous/mvrest?path=/eosm-public-offer&officeLabels=%7B%7D&page=1&pageSize=100000&sortColumn=zdatzvsm&sortOrder=-1'
-
-    req = Request.from_url(
-        url=url,
+async def test_long_request(request_queue_apify: RequestQueue) -> None:
```

Comment:
Why did you rename the test? I see a difference in "long URL" and "long request" :slightly_smiling_face: